plugins/modules/k8s_drain.py

```diff
@@ -64,6 +64,11 @@
             - Ignore DaemonSet-managed pods.
             type: bool
             default: False
+        delete_emptydir_data:
+            description:
+            - Continue even if there are pods using emptyDir (local data that will be deleted when the node is drained)
+            type: bool
+            default: False
```

Comment:
```suggestion
            default: False
            version_added: '2.3.0'
```

Reply:
Added `version_added: 2.3.0` (without `'`), in line with the rest of the block